packages/react-barcode-scanner/src/hooks/use-camera.tsx

```diff
@@ -18,39 +18,45 @@ const DEFAULT_CONSTRAINTS: MediaTrackConstraints = {
  * Manage camera stream state.
  * @param ref a RefObject of HTMLVideoElement
  * @param trackConstraints a MediaTrackConstraints object, provide advanced options
- * @returns a boolean that indicates whether the camera is supported
+ * @returns {object} { isCameraReady, error }
+ *   - isCameraReady: Whether the camera is ready
+ *   - error: Error object
  * @example
  * import { type RefObject } from 'react'
  * import { useCamera } from 'react-barcode-scanner'
  *
  * function App () {
  *   const ref = useRef<HTMLVideoElement>(null)
- *   const [isCameraSupported] = useCamera(ref)
+ *   const { isCameraReady, error } = useCamera(ref)
  *
  *   useEffect(() => {
- *     if (!isCameraSupported) {
- *       console.log('[react-barcode-scanner]: Camera is not supported')
```

Comment:
`useCamera` would return `isCameraSupported=false` by default - IMO it makes more sense to return a flag indicating the if camera is ready instead